graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -35,7 +36,10 @@ const ShowContentPackPage = createReactClass({
 
 
   componentDidMount() {
-    ContentPacksActions.get(this.props.params.contentPackId);
+    ContentPacksActions.get(this.props.params.contentPackId).catch(() => {
+      UserNotification.error('Cannot find a matching Content Pack.');
```

Comment:
In case the backend returns any other error than a 404, we will tell the user we couldn't find the content pack, where that is not really what happened. I think we should check the error status is a 404 before doing that.